text/0002-modules.md

```diff
@@ -26,17 +26,18 @@ that there is only one network request for a whole group (bundle) of build-time
 [Browser support for modules](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#Browser_compatibility)
 started in Chrome 61, Firefox 60, Edge 16, and Safari 10.1. Internet Explorer does not support modules.
 
-A module loader is a javascript library that allows you to use modules in a browser that does not support them. This makes
-it possible for us to use modules in Internet Explorer.
+A [module loader](https://www.jvandemo.com/a-10-minute-primer-to-javascript-modules-module-formats-module-loaders-and-module-bundlers/) is
+a javascript library that allows you to use modules in a browser that does not support them. This makes it possible for us to use modules
```

Comment:
I do not see this as a big deal for us. 😊 
We have the freedom to specify which browsers we do support.

Reply:
Oo really? I have been planning on adding a PR for browser support, but was working under the assumption that we wouldn't have much control over that because of how widespread OpenMRS is being used. Are people currently using IE with OpenMRS? Would removing IE mean that fewer people can use it?

Reply:
Also, I just pushed a commit that clarifies that using SystemJS is not just for IE11. There are different features of modules and not even Chrome/Firefox/Edge/Safari have one of the important ones yet -- import maps.

Reply:
If it is more than just IE, then that is fine.